php/custom.ini

```diff
@@ -11,6 +11,10 @@ post_max_size = "100M"
 [date]
 date.timezone = "Europe/Paris"
 
+[session]
+session.cookie_secure = ""
```

Comment:
This setting expects a boolean value and is disabled by default ([source](http://php.net/manual/fr/session.configuration.php#ini.session.cookie-secure)).